src/CheckRightsService/Startup.cs

```diff
@@ -9,7 +9,7 @@
 using LT.DigitalOffice.CheckRightsService.Models;
 using LT.DigitalOffice.CheckRightsService.Repositories;
 using LT.DigitalOffice.CheckRightsService.Repositories.Interfaces;
-using LT.DigitalOffice.CheckRightsService.Validator;
+using LT.DigitalOffice.CheckRightsService.Validators;
```

Comment:
Triple work.

Reply:
I know, but that was annoying, hah

Reply:
Oops, the service doesn't run without these changes